pkgs/top-level/all-packages.nix

```
@@ -19711,6 +19711,7 @@ in
   swaylock = callPackage ../applications/window-managers/sway/lock.nix { };
 
   swaylock-fancy = callPackage ../applications/window-managers/sway/lock-fancy.nix { };
+  swaylock-effects = callPackage ../applications/window-managers/sway/lock-effects.nix { };
```

```suggestion

  swaylock-effects = callPackage ../applications/window-managers/sway/lock-effects.nix { };
```

`swaylock-fancy` isn't right below `swaylock`, so no need (IMHO) for `swaylock-effects` to be right under `swaylock-fancy`

---

And I'd also put `-effects` before `-fancy`, just to keep it sorted alphabetically.